index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microvers</h1> 
```

Comment:
- [ ] It is required to create a class for the header and use the class name to style the content of the header.

### Project Requirement
![style](https://github.com/axcell-0/hello-microverse/assets/26597432/f52689d3-974e-4d68-8201-bd3d61c5a94a)